config/default.yml

```diff
@@ -581,6 +581,10 @@ Layout/ExtraSpacing:
   # things with the previous or next line, not counting empty lines or comment
   # lines.
   AllowForAlignment: true
+  # When true, allows things like 'obj.meth(arg)  # comment',
+  # rather than insisting on 'obj.meth(arg) # comment'.
+  # If done for alignment, either this OR AllowForAlignment will allow it.
+  AllowForTrailingComments: false
```

Comment:
I think `AllowBeforeTrailingComments` would be more specific, as the current one doesn't really indicate where the spaces are allowed. WDYT?

Reply:
Sounds good to me!